scripts/build_synapse_cdn_webacl.sh

```diff
@@ -5,16 +5,19 @@
 
 set +x
 
-# dev, staging or prod
+# dev or prod
```

Comment:
Should this not be "dev, tst, or prod"?

Reply:
The stack is the account it runs in (dev or prod). The instance references the 'fixed' instances (what I call alias),
that is prod, staging or tst (we have a CDN for each of them).

Reply:
> The stack is the account it runs in (dev or prod). 
Shall we change the STACK paramter name to AWS_ACCOUNT or something else that's more descriptive?

> The instance references the 'fixed' instances (what I call alias)
I'm getting confused.  Again, is there another, more descriptive term to use here?

> it would make more sense to add it to the CDN template itself and delete all the CdnWebAcl code
Are you saying to hold off on this PR while you revisit the stack builder?